src/awscdk/github.ts

```diff
@@ -75,7 +113,7 @@ export class GithubCDKPipeline extends CDKPipeline {
 
     this.deploymentWorkflow.addJob('synth', {
       name: 'Synth CDK application',
-      runsOn: ['ubuntu-latest'],
+      runsOn: this.options.runnerTags ?? ['ubuntu-latest'],
```

Comment:
Why not make this thing a const at the top of the class?

or at least make 'ubuntu-latest' a const so we can change it easily? ;)